src/ir/module-splitting.cpp

```diff
@@ -146,10 +146,21 @@ void TableSlotManager::addSlot(Name func, Slot slot) {
 }
 
 TableSlotManager::TableSlotManager(Module& module) : module(module) {
-  if (module.features.hasReferenceTypes()) {
+  Export* emscripten_table_export =
+    module.getExportOrNull("__indirect_function_table");
+  Table* indirect_table =
+    module.tables.size() == 1 ? module.tables[0].get() : nullptr;
+  bool emscripten_import = indirect_table->imported() &&
```

Comment:
```suggestion
  bool emscriptenTableImport = indirect_table->imported() &&
```
(this also adds "Table" to the name to match the other one)